internal/connector/connector_test.go

```diff
@@ -143,6 +126,31 @@ func TestDeployConcurrent_ConnectorsAndPluginsWithDifferentModules(t *testing.T)
 		},
 	}
 
+	_, err := exec.LookPath("fio")
+	if err != nil {
+		logger.Warningf("Did not find Flexible IO Tester, 'fio', on system path. Please install it.")
+	} else {
+		logger.Infof("Found 'fio' on system path, adding fio plugin to test suite")
```

Comment:
Since we want this test to run `fio`, it makes good sense to issue a warning if it is not found; however, logging the fact that we found it is just noise.  The fact that there is no warning should be sufficient (not to mention the fact that adding it to the test suite will presumably result in other evidence of its presence).

Having verbose logging is fine during development; but, in terms of what gets merged, we should give some thought to what is actually useful in the general case and avoid creating useless clutter.

Reply:
Yeah. Either remove the found log, or make it debug.
Also, it entirely skips the assignment done. Is this intentional? Is just a warning enough?

Reply:
okay it's gone